Demo/app/main.py

```diff
@@ -14,6 +14,13 @@
 from app.core.finding_deduplication import FindingDeduplication
 from app.core.final_evaluation import FindingEvaluator
 
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+max_findings_per_submission = int(os.getenv("MAX_FINDINGS_PER_SUBMISSION", 20))
```

Comment:
Please add this under config.py as `MAX_FINDINGS_PER_SUBMISSION` and then import from there.